airbyte-integrations/bases/standard-destination-test/src/main/java/io/airbyte/integrations/standardtest/destination/DestinationAcceptanceTest.java

```diff
@@ -96,7 +99,6 @@
 public abstract class DestinationAcceptanceTest {
 
   private static final Random RANDOM = new Random();
-  private static final String NORMALIZATION_VERSION = "dev";
```

Comment:
are we no longer overriding the version with `dev`? (if yes: that's a problem, because it will be _really_ difficult to test a PR that modifies a destination connector _and_ normalization)

Reply:
Thank you for your comments. This is my shortcoming. Instead of the dev version, I started taking the latest version from the definition file. Changed it and double checked that everything works fine